leakcanary-android-core/src/main/java/leakcanary/WorkManagerHeapAnalyzer.kt

```diff
@@ -0,0 +1,50 @@
+package leakcanary
+
+import android.os.HandlerThread
+import androidx.work.OneTimeWorkRequest
+import androidx.work.OutOfQuotaPolicy
+import androidx.work.WorkManager
+import leakcanary.EventListener.Event
+import leakcanary.EventListener.Event.HeapDump
+import leakcanary.internal.HeapAnalyzerWorker
+import leakcanary.internal.HeapAnalyzerWorker.Companion.asWorkerInputData
+import leakcanary.internal.InternalLeakCanary
+
+/**
+ * When receiving a [HeapDump] event, starts a WorkManager worker that performs heap analysis.
+ */
+object WorkManagerHeapAnalyzer : EventListener {
```

Comment:
@ItzNotABug thanks for #2197 ! It was super helpful to get this done.

Reply:
Great! Happy to help. :)